fixtures/consumer/__snapshots__/build.test.ts.snap

```diff
@@ -12,6 +12,20 @@ exports[`build side-effects from dist 1`] = `
   "dist/side-effects/lib/css/reset/index.mjs",
   "dist/side-effects/lib/css/reset/resetTracker.cjs",
   "dist/side-effects/lib/css/reset/resetTracker.mjs",
+  "dist/side-effects/lib/playroom/FrameComponent.cjs",
```

Comment:
Looks like you need to do a `pnpm install` locally and rerun the test.

Reply:
Interesting. I did that before opening the PR and it still wanted to update the snapshot, so I thought I'd see what CI thought. Let me play around more 🤔

Reply:
Correction: [`71efb30` (#1240)](https://github.com/seek-oss/braid-design-system/pull/1240/commits/71efb30dc981d44bb69c9ec6a975d1afb3bf9556)